source/docs/essentials/add-new-attribute.md

```diff
@@ -0,0 +1,156 @@
+
+If we want our code to be taken into account when the module is loaded, we must index it.
+
+Create a file named `index.js` and type in the following code. It indexes the schema and the resolver from earlier.
+
+```js
+// mymodule/server/modules/acme/index.js
+import typeDefs from "./schema.gql";
+import resolvers from "./resolvers";
+
+export default {
+  namespace: "Acme",
+  typeDefs: typeDefs,
+  resolvers: resolvers
+};
+```
+
+### Overriding the files
+
+In **this** example, the files to be overridden are `ProductSynthesisFragment.gql` and `Synthesis.js` both located at `node_modules/front-commerce/src/web/theme/modules/ProductView/Synthesis/`. Copy and paste them into your folder.
+
+### Making the query
+
+In `ProductSynthesisFragment;gql`, add the following query line. It means that the application will request the `rate` field as well when sending the query. (Depending on the version of Front-Commerce you are using, the content might differ slightly.)
+
+```diff
+#import "theme/components/organisms/Configurator/ProductConfiguratorFragment.gql"
+#import "theme/modules/AddToCart/ProductStockFragment.gql"
+#import "theme/components/atoms/Typography/Price/ProductPriceFragment.gql"
+#import "theme/modules/ProductView/Details/ProductDetailsFragment.gql"
+
+fragment ProductSynthesisFragment on Product {
+  sku
+  name
+  ...ProductPriceFragment
+  description
++rate
+  ...ProductConfiguratorFragment
+  ...ProductStockFragment
+  ...ProductDetailsFragment
+  configurations {
+    product {
+      sku
+      ...ProductPriceFragment
+    }
+  }
+}
+```
+
+### Displaying the result on screen
+
+We're almost there! In the file `Synthesis.js` (which generates the product page), you can add some code to display the `rate`. Knowing React.js or at least HTML syntax is necessary to perform this step without screwing the other items on the page.
+
+```js
+// Many more things here
+const ProductSynthesis = props => {
+  const {
+    product
+    // Many more things here
+  } = props;
+  return (
+    <div className="product__synthesis">
+      {/* Many more things here */}
+      <p>Rate: {product.rate} / 5</p>
+      {/* Many more things here */}
+    </div>
+  );
+};
+
+export default ProductSynthesis;
+```
+
+## Add the module
+
+Last step: adding the module we just made to Front-Commerce. Edit the `.front-commerce.js` file (root folder) as shown below to do so:
+
+```diff
+module.exports = {
+  name: "Front Commerce Skeleton",
+  url: "http://localhost:4000",
+-modules: ["./src"],
++modules: ["./mymodule", "./src"],
+  serverModules: [
+    { name: "FrontCommerce", path: "server/modules/front-commerce" },
+    { name: "Magento2", path: "server/modules/magento2" },
++{ name: "Acme", path: "./mymodule/server/modules/acme" }
+  ]
+};
+
+```
+
+## Additional note
+
+This page has been redacted by an intern, who is also a rookie in React, GraphQL, Front-Commerce, and Web development in general. Please, don't be mad at me, I did my best! :)
```

Comment:
haha nobody will! Don't worry about this. It's already great and will become even greater. People will be grateful that you've helped them along the way :)

I'm not quite sure we should keep this final note, though. Even if it's cute!